src/main/java/org/generationcp/middleware/service/api/KeySequenceRegisterService.java

```diff
@@ -55,4 +58,18 @@
 	 * @param lastSequenceUsed
 	 */
 	void saveLastSequenceUsed(String keyPrefix, Integer lastSequenceUsed);
+
+	/**
+	 * Returns the List of KeySequenceRegister given the Key Prefixes
+	 *
+	 * @param keyPrefixes
+	 */
+	List<KeySequenceRegister> getKeySequenceRegistersByPrefixes(List<String> keyPrefixes);
```

Comment:
The parameter is self-descriptive. We can just name as `getKeySequences`

Reply:
Based on logic simplification in BreedingManager [PR](https://github.com/IntegratedBreedingPlatform/BreedingManager/pull/337), we possibly don't need this method `getKeySequences`